rpc/events.go

```diff
@@ -44,14 +50,27 @@ type EventsChunk struct {
 	ContinuationToken string          `json:"continuation_token,omitempty"`
 }
 
+type SubscriptionID struct {
+	ID uint64 `json:"subscription_id"`
+}
+
+func (s *SubscriptionID) UnmarshalJSON(data []byte) error {
```

Comment:
why do we need custom `UnmarshalJSON` ?

Reply:
We need subscription id as a param to `SubscriptionNewHeads`

Reply:
deleted